src/mds/MetricAggregator.cc

```diff
@@ -32,6 +63,15 @@ void MetricAggregator::ping_all_active_ranks() {
 int MetricAggregator::init() {
   dout(10) << dendl;
 
+  std::string labels = ceph::perf_counters::key_create("mds_client_metrics",
+						       {{"fs_name", mds->mdsmap->get_fs_name()},
+							{"id", stringify(mds->get_global_id())}});
+  PerfCountersBuilder plb(m_cct, labels, l_mds_client_metrics_start, l_mds_client_metrics_last);
+  plb.add_u64(l_mds_client_metrics_num_clients,
+	      "num_clients", "Numer of client sessions", "mcli", PerfCountersBuilder::PRIO_CRITICAL);
```

Comment:
number of client sessons? typo maybe?

Reply:
@julpark-rh Yes it's a typo. Will fix it.